src/app/pages/news/13.1/06-market.md

```diff
@@ -0,0 +1,21 @@
+## Marketplace Website {#market}
```

Comment:
I'd merge website promotion and market-artifacts. IMHO these rather large text-blocks overemphasizes the increment accomplished in 13.1 ... and distracts from the more meaningful changes :man_shrugging: